demisto_sdk/commands/common/MDXServer.py

```diff
@@ -83,13 +96,14 @@ def start_docker_MDX_server(
     try:
         yield True
     finally:
-        stop_docker_container(container)
+        remove_container(container)
 
 
-def stop_docker_container(container):
+def remove_container(container):
     if container:
-        logging.info("Stopping mdx docker server")
-        container.stop()  # type: ignore
+        print(f"Removing container {container.name}")
+        container.remove(force=True)
```

Comment:
If the container isn't there, will `remove` fail, or be quiet?

Reply:
as long as you have container object, it exists, hence it wouldn't fail, but if docker api returned and error then an exception will be raised. In our case it can raise only if container doesn't exist.

https://docker-py.readthedocs.io/en/stable/containers.html#docker.models.containers.Container.remove